plugins/module_utils/ansible_freeipa_module.py

```diff
@@ -103,7 +103,7 @@ def parse(version_str):
 
     import socket
     import base64
-    import six
+    from ansible.module_utils import six
```

Comment:
This should also be added to the `new_module` templates (in another PR).

Reply:
Done